### PR TITLE
Add missing fields to PGN 127489 parser

### PR DESCRIPTION
**Pull request by fabdrol**
Add the following (missing) fields to the test & parser for PGN 127489:

- Fuel Pressure
- Engine coolant pressure
- Total engine hours
- Engine oil temperature

Reference & related issue:

1. http://www.nmea.org/Assets/july%202010%20nmea2000_v1-301_app_b_pgn_field_list.pdf
2. https://github.com/SignalK/n2k-signalk/issues/117

### CODE REVIEW

**Comment by fabdrol** [comment]
@tkurki we came across the missing fields when working on parsing NMEA2000 engine PGNs coming from an engine through a Maretron J1939 to N2K converter. This PR implements the missing fields, as per the NMEA reference document found online.

File: test/127489_engine_params.js [hunk 1]
```diff
@@ -97,24 +113,5 @@ describe('127489 engine parameters Starboard', function () {
       'notifications.propulsion.starboard.maintenanceNeeded.value.state',
       'alarm')
     tree.should.be.validSignalKVesselIgnoringIdentity
-
```
Comment (tkurki) [inline test/127489_engine_params.js]: Why remove this test?

Comment (fabdrol) [reply test/127489_engine_params.js]: I thought I copied a bit of code twice, so removed the duplicate. Didn't realise it was a discrete test ;)

**Comment by fabdrol** [comment]
@tkurki separating the tests works for me. The data "sourcing" is mixed: some of it is from our real world data but not all, as not every field is present. So in order to implement the remaining missing fields I added them to the JSON myself.

**Comment by fabdrol** [comment]
@tkurki there you go.

**Pull request merged**